application/src/main/java/org/thingsboard/server/service/install/SqlDatabaseUpgradeService.java

```diff
@@ -768,10 +768,10 @@ public void upgradeDatabase(String fromVersion) throws Exception {
                         } catch (Exception e) {
                         }
 
-                        conn.createStatement().execute("UPDATE tb_schema_settings SET schema_version = 3005002;");
-                        log.info("Schema updated to version 3.5.2.");
+                        conn.createStatement().execute("UPDATE tb_schema_settings SET schema_version = 3006000;");
+                        log.info("Schema updated to version 3.6.0.");
                     } else {
-                        log.info("Skip schema re-update to version 3.5.2. Use env flag 'SKIP_SCHEMA_VERSION_CHECK' to force the re-update.");
+                        log.info("Skip schema re-update to version 3.6.0. Use env flag 'SKIP_SCHEMA_VERSION_CHECK' to force the re-update.");
```

Comment:
I am going to merge this now. Please improve and refactor this code to something like this:

```
upgradeDb(3005001, 3006000, Consumer<Connection>);
and one more:
executeStatement(conn, "ALTER TABLE rule_node ADD COLUMN IF NOT EXISTS configuration_version int DEFAULT 0;")
```